user-operator/src/main/java/io/strimzi/operator/user/operator/SimpleAclOperator.java

```diff
@@ -217,6 +217,12 @@ public SimpleAclOperator(Vertx vertx, kafka.security.auth.SimpleAclAuthorizer au
         return result;
     }
 
+    private Iterator<Tuple2<Resource, scala.collection.immutable.Set<Acl>>> getResourceAclsIterator(scala.collection.immutable.Map<Resource, scala.collection.immutable.Set<Acl>> rules) {
```

Comment:
What's wrong with the name `resourceAclsIterator()`? Why do you feel compelled to prefix it with `get`? It's not a getter in the JavaBean sense. And obviously it "gets" something (which depends on its parameter) because it returns non-`void`?

Reply:
Updated